### PR TITLE
Mail cleanup

### PR DESCRIPTION
**Pull request by TheRittler**
added logging and some cleaning and optimizing.

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailSender.java [hunk 1]
```diff
@@ -18,5 +17,5 @@
      * @param subject       the subject of the mail
      * @param message       the content of the mail
      */
-    public void sendMail(String recipientMail, String subject, String message) throws MessagingException, UnsupportedEncodingException;
+    void sendMail(String recipientMail, String subject, String message) throws MessagingException, UnsupportedEncodingException;
```
Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailSender.java:20]: Any reason for this?

Comment (TheRittler) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailSender.java:20]: As far as I know modifiers are useless in interfaces.

Comment (TheRittler) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailSender.java:20]: A public modifier implicates, that the Method has a modifier, but it hasn't

Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailSender.java:20]: Oh, it's the interface, fine then.

**Pull request merged**